doc/source/appendix/ontologies.rst

```diff
@@ -38,30 +39,34 @@ What is the minimum attribute requirement for  OntologyTerm in GA4GH?
 
 Conceptually (and consistent with the metadata branch)
 
-:ontologyId:
-  required and implemented as URI
-  we assume this resolves to a meaningful document, e.g. http://purl.obolibrary.org/obo/SO_0000147
+:termId:
+  The unique identifier of the term in an ontology source. This should be a CURIE (e.g., SO:0001234), but may be an alphanumerical string (e.g., 8000/3) if no CURIE are available.
 :term:
   preferred but not required (e.g. ‘exon’); corresponds to class label
+:termUri:
+  A fully qualified URI that can be used to access details regarding the term. e.g. http://purl.bioontology.org/ontology/SNOMEDCT/76345009.
 :sourceName:
   not required since should be resolved from prefix etc., but supporting/fall-back in case of non-standard/deprecated/entropic annotations; possible use for CURIEs in compact sequence ontology implementations  (e.g. SO:0000147)
 :sourceVersion:
-  not required but good practice; if no explicit versioning, ISO8601 formatted data of retrieval should be used
+  not required but good practice; if no explicit versioning, ISO8601 formatted data of retrieval should be used.
```

Comment:
In some cases, there are versioned artifacts; it may be good to point out that these URIs too could be used in this field.eg.  http://purl.obolibrary.org/obo/uberon/releases/2016-01-26/uberon.owl